pkg/reconcilermanager/controllers/gitsync_env.go

```diff
@@ -212,21 +210,21 @@ func gitSyncEnvs(_ context.Context, opts options) []corev1.EnvVar {
 	}
 	result = append(result, corev1.EnvVar{
 		Name:  gitSyncPeriod,
-		Value: fmt.Sprintf("%f", opts.period),
+		Value: fmt.Sprintf("%fs", opts.period),
```

Comment:
It looks like git-sync expects a `time.Duration` format string here. In `gitSyncEnvs` we convert the `metav1.Duration` -> `float64` (seconds) and then here we are formatting that float into string seconds.

I suggest refactoring this so that the `options` struct accepts a `time.Duration` type for `period` and then we can just use the builtin String conversion

Reply:
https://github.com/GoogleContainerTools/kpt-config-sync/blob/main/pkg/reconcilermanager/controllers/rootsync_controller.go#L686

Reply:
done